CHANGELOG.md

```diff
@@ -61,6 +61,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ### Changed
 
+- Enabled statfs FS type magic constants for `target_os = "android"`
```

Comment:
Can you put this in the "Added" section instead? We try to keep "Changed" to just breaking changes.

Reply:
Sure.